omod/src/main/java/org/openmrs/module/spa/servlet/SpaServlet.java

```diff
@@ -17,13 +22,25 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import static org.openmrs.module.spa.filter.SpaFilter.DEFAULT_SPA_BASE_URL;
+import static org.openmrs.module.spa.filter.SpaFilter.GP_KEY_SPA_BASE_URL;
+
 public class SpaServlet extends HttpServlet {
 
 	private static final long serialVersionUID = 1L;
+	protected final Log log = LogFactory.getLog(getClass());
```

Comment:
This is our convention for logging: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-loggingLogging

Reply:
@dkayiwa see changes and whether it conforms